### PR TITLE
Change icon glyphs to match iconochive

### PR DESCRIPTION
**Pull request by pezvi**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,3 +11,4 @@ node_modules/
 /lib/
 
 *.log
+.DS_Store
```
Comment (jbuckner) [inline .gitignore:14]: I would recommend adding this to your local global `.gitignore` file. It's going to keep popping up in every repo you touch 😅 You can do this:

Create a file in your home directory `~/.gitignore`. Add this to it:
```
.DS_Store
```

Configure your global gitignore file:
```
git config --global core.excludesfile ~/.gitignore
```
Then you can add OS-specific stuff like this to it.

**Review by jbuckner — APPROVED** [review]
One suggestion, but looks good otherwise 👍

File: packages/icon-share/share.svg [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
   </g>
 </svg>
+
+
```
Comment (iisa) [inline packages/icon-share/share.svg:13]: suggestion: remove extra space

File: icons/share.svg [hunk 3]
```diff
@@ -1,12 +1,11 @@
...
   </g>
-</svg>
+</svg>
```
Comment (iisa) [inline icons/share.svg:11]:
```suggestion
</svg>

```

File: icons/search.svg [hunk 4]
```diff
@@ -8,5 +8,5 @@
...
+    d="m17.0555551 41.3194459c0-12.7430552 10.3541664-23.1027772 23.0847216-23.1027772 12.7166664 0 23.0777773 10.359722 23.0777773 23.1027772 0 12.7361108-10.3611109 23.0986106-23.0777773 23.0986106-12.7305552 0-23.0847216-10.3624998-23.0847216-23.0986106zm-17.24305512 0c0 22.2916661 18.04583292 40.3472213 40.32777672 40.3472213 8.9208332 0 17.145833-2.9319449 23.8194439-7.8527776l24.1513883 24.0777771c1.2125 1.1402778 2.8430555 1.8430556 4.6374999 1.8430556 3.7444443 0 6.7805554-3.0361111 6.7805554-6.7791665 0-2.0652778-.9222222-3.9069444-2.3736111-5.1499999l-23.718055-23.7458328c4.4152777-6.4791665 7.0152776-14.3055552 7.0152776-22.7402772 0-22.2791661-18.0458328-40.34861006-40.312499-40.34861006-22.2819438 0-40.32777672 18.06944396-40.32777672 40.34861006z" />
 </svg>
```
Comment (iisa) [inline icons/search.svg:12]:
```suggestion
</svg>

```

**Review by iisa — APPROVED** [review]

**Pull request merged**